src/main/java/org/eclipse/microprofile/starter/rest/APIEndpointV7.java

```diff
@@ -0,0 +1,8 @@
+package org.eclipse.microprofile.starter.rest;
+
+import javax.ws.rs.Path;
+
+@Path("/7")
```

Comment:
Based on my understanding from @rdebusscher , there is no need to do this change.